doc/js_tutorials/js_assets/js_dnn_example_helper.js

```diff
@@ -0,0 +1,141 @@
+getBlodFromImage = function(inputSize, mean, std, swapRB, canvasId) {
```

Comment:
Blod?

Reply:
Sorry for this mistake, I'll correct it.